Tasks/AppCenterDistributeV3/Tests/L0SymAPPXSYMs_single.ts

```diff
@@ -16,24 +16,11 @@ let tmr: tmrm.TaskMockRunner = new tmrm.TaskMockRunner(taskPath);
 
 tmr.setInput('serverEndpoint', 'MyTestEndpoint');
 tmr.setInput('appSlug', 'testuser/testapp');
-tmr.setInput('app', '/test/path/to/my.ipa');
+tmr.setInput('app', 'C:/path/to/my.appxbundle');
```

Comment:
Why is this changed to absolute path? Is that how it works for windows builds?

Reply:
No, it still uses relative one, actually test do nothing with actual file, so it a nice catch and should be fixed :)

Reply:
Thanks for pointing out! Fixed.